### ISSUE TITLE
Fatal error: Unable to sync 'optimism'. Block range is too large

### ISSUE THREAD

**Issue by stepandel** [ponder-sh/ponder#1283]
While setting up ponder for EAS on `optimism` running into `Block range is too large` error. This happens while backfilling from `124380685` block using Alchemy rpc

There seems to be no way to set block range in settings.

Full error
```
Fatal error: Unable to sync 'optimism' from 124411814 to 124414326.
RpcRequestError: RPC Request failed.

URL: https://mainnet.optimism.io
Request body: {"method":"eth_getLogs","params":[{"fromBlock":"0x76a5fa6","toBlock":"0x76a6976","topics":[["0xf930a6e2523c9cc298691873087a740550b8fc85a0680830414c148ed927f615"]],"address":"0x42cdc5d4b05e8dacc2fcd181cbe0cc86ee14c439"}]}

Details: Block range is too large
```

**Comment by 0xOlias** [comment]
Are you sure you're using Alchemy? The error message prints `https://mainnet.optimism.io`.

I did confirm that they're now using a new error message, and updated our retry logic to handle it (https://github.com/ponder-sh/ponder/pull/1284). Will ship shortly.

FWIW, we've found that specific URL to be extremely unreliable (the most inconsistent behavior we've ever seen 😅). I'd strongly recommend using a different RPC.
